src/pages/es/guides/typescript.md

```diff
@@ -11,26 +11,15 @@ Astro no realiza ninguna verificación de tipos por sí mismo. La verificación
 
 ## Configuración
 
-Se **recomienda encarecidamente** que crees un archivo `tsconfig.json` para que las herramientas como Astro y VSCode sepan cómo interpretar tu proyecto. Algunas funcionalidades (como las importaciones de paquetes npm) no son totalmente compatibles con TypeScript a menos que crees un archivo `tsconfig.json`.
+[] incluye un archivo `tsconfig.json` en tu proyecto. Incluso si tu no escribes código TypeScript, este archivo es importante pues así, herramientas como Astro y VS Code saben como entender tu proyecto. Algunas características(como importación de archivos npm) no están totalmente soportadas en el editor sin un archivo `tsconfig.json`. Si instalas Astro manualmente, asegurate de create este archivo por tú mismo.
```

Comment:
```suggestion
Los *starter projects* de Astro incluyen un archivo `tsconfig.json` en tu proyecto. Incluso si no escribes código TypeScript, este archivo es importante para que herramientas como Astro y VS Code sepan cómo interpretar tu proyecto. Algunas características (como importaciones de paquetes npm) no están totalmente soportadas en el editor sin un archivo `tsconfig.json`. Si instalas Astro manualmente, asegúrate de crear este archivo por ti mismo.
```

Reply:
Thanks for the recommendations. I will implement the suggested updates.

P.S. That `[]` my dear Lord...